debug_toolbar/middleware.py

```diff
@@ -78,6 +79,8 @@ def __init__(self, get_response):
             markcoroutinefunction(self)
 
     def __call__(self, request):
+        # Start tracking middleware execution time
+        middleware_start_time = perf_counter()
```

Comment:
We probably want to rename this to `toolbar_` and update the text for it to be specific to the toolbar. Middleware may imply all of the middlewares together.

Reply:
Yes, that makes sense. I have renamed it.